prow/cluster/hook_deployment.yaml

```diff
@@ -30,10 +30,10 @@ spec:
       labels:
         app: hook
     spec:
-      terminationGracePeriodSeconds: 30
+      terminationGracePeriodSeconds: 180
```

Comment:
SIGTERM is [ignored by hook](https://github.com/kubernetes/test-infra/blob/d9dc4f7fd063da19db7539b8017407b43e8d07a7/prow/cmd/hook/main.go#L89-L92), but it doesn't do any graceful shutdown.